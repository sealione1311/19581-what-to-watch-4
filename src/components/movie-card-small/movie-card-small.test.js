import React from 'react';
import renderer from 'react-test-renderer';
import MovieCardSmall from "./movie-card-small.jsx";

const film = {
  id: 1,
  title: `Bohemian Rhapsody`,
  smallImage: `img/bohemian-rhapsody.jpg`
};

describe(`Render MovieCardSmall`, () => {
  it(`Render MovieCardSmall`, () => {
    const tree = renderer.create(
        <MovieCardSmall
          movie = {film}
          onTitleClick = {() => {}}
          onCardHover = {() => {}}
          onSmallCardClick={() => {}}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
