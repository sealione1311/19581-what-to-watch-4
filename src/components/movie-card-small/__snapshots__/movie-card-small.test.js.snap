// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render MovieCardSmall Render MovieCardSmall 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  id={1}
  onClick={[Function]}
  onMouseOut={[Function]}
  onMouseOver={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <video
      className="video-player"
      height="175"
      muted="muted"
      poster="img/bohemian-rhapsody.jpg"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      Bohemian Rhapsody
    </a>
  </h3>
</article>
`;
